src/index.ts

```diff
@@ -2,21 +2,21 @@ import { SignedWrappedDocument, v2, v3, WrappedDocument } from "@govtechsg/open-
 import { verificationBuilder } from "./verifiers/verificationBuilder";
 import { Verifier, Verifiers } from "./types/core";
 import { openAttestationHash } from "./verifiers/documentIntegrity/hash/openAttestationHash";
-import { Identity, openAttestationDnsTxt } from "./verifiers/issuerIdentity/dnsText/openAttestationDnsTxt";
 import { isValid } from "./validator";
-import { openAttestationEthereumTokenRegistryStatus } from "./verifiers/documentStatus/tokenRegistryStatus/openAttestationEthereumTokenRegistryStatus";
-import { openAttestationEthereumDocumentStoreStatus } from "./verifiers/documentStatus/documentStoreStatus/openAttestationEthereumDocumentStoreStatus";
-import { OpenAttestationDidSignedDocumentStatus } from "./verifiers/documentStatus/didSignedDocumentStatus";
-import { OpenAttestationDidSignedDidIdentityProof } from "./verifiers/issuerIdentity/didIdentityProof";
-import { OpenAttestationDnsDid } from "./verifiers/issuerIdentity/dnsDidProof";
+import { openAttestationEthereumTokenRegistryStatus } from "./verifiers/documentStatus/tokenRegistry";
+import { openAttestationEthereumDocumentStoreStatus } from "./verifiers/documentStatus/documentStore";
+import { openAttestationDidSignedDocumentStatus } from "./verifiers/documentStatus/didSigned";
+import { Identity, openAttestationDnsTxtIdentityProof } from "./verifiers/issuerIdentity/dnsTxt";
+import { openAttestationDidIdentityProof } from "./verifiers/issuerIdentity/did";
+import { openAttestationDnsDidIdentityProof } from "./verifiers/issuerIdentity/dnsDid";
 
 const openAttestationVerifiers: Verifiers[] = [
```

Comment:
what's the convention followed for the verifiers name ?

Reply:
All the identityProof ones are not postfixed with `IdentityProof` while document status will be postfixed by `Status`

Reply:
before that we have DnsText, but that was the only one, so we added more now, all of them are postfixed with IdentityProof to distinguish